Clarify how certificate fingerprints work for Fleet Server (#5138)
elastic/docs-content: reference/fleet/certificates-rotation.md

1. In {{fleet}} open the **Settings** tab.
2. In the **Outputs** section, click the edit button for the {{es}} output that requires a certificate rotation.

> 3. In the **Elasticsearch CA trusted fingerprint** field, add the new trusted fingerprint to use. This is the SHA-256 fingerprint (hash) of a certificate authority that is present in the certificate chain sent by {{es}} during the TLS handshake.

Reviewer: The "`of`" and passive voice references here make it sound a tad bit confusing, almost like it's a handful to digest. You can ignore if this suggestion changes the meaning. 
```suggestion
3. In the `Elasticsearch CA trusted fingerprint` field, add the new trusted fingerprint to use. This is the SHA-256 fingerprint's (hash) certificate authority that's present in the certificate chain {{es}} sends during the TLS handshake.
```

Author: See my previous comment about the field. As for "The SHA-256 fingerprint's (hash) certificate authority” - this is not correct because we’re talking about a certificate authority which can be hashed in order to get the certificate authority’s fingerprint.

```suggestion
3. In the **Elasticsearch CA trusted fingerprint** field, add the new trusted fingerprint to use. This is the SHA-256 fingerprint (hash) of a certificate authority that's present in the certificate chain {{es}} sends during the TLS handshake.
```
